GETTING_STARTED.ipynb

```diff
@@ -17,8 +17,29 @@
    "metadata": {},
```

Comment:
<p>For Seaborn you have this extra comment:</p><p> </p><p><code># Uncomment to install if required</code></p><p> </p><p>Is that something we should add here too? Or maybe even just keep it always visible, as it is a no-op if it's already installed?</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/home-assistant/home-assistant-notebooks/pull/22/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
If @frenck is able to add seaborn we can remove this comment, otherwise I could just uncomment as you say

Reply:
Seaborn is part of the next add-on release, so I'm fine with leaving it like this.